main.go

```diff
@@ -117,3 +74,13 @@ func main() {
 		fmt.Fprintf(os.Stderr, "%v\n", err)
 	}
 }
+
+func getExecPath() string {
+	ex, err := os.Executable()
+	if err != nil {
+		fmt.Println(err)
+		return ""
```

Comment:
This function should just return an error.  As it stands, this will lead to more confusing errors downstream (since filepaths will not be as expected).  Neither use of `getExecPath` even checks the output against the empty string.